HpToolsLauncher/Launcher.cs

```diff
@@ -110,6 +111,11 @@ public static string DateFormat
             get { return _dateFormat; }
         }
 
+        public static string DefaultBranches
+        {
+            get { return _defaultBranches; }
+        }
+

```

Comment:
is this part is necessary, this file is relevant to UFT test executions? can you please check your code is working if you remove your code from here?